etc/containerpilot.json

```diff
@@ -1,43 +1,43 @@
 {
   "consul": "{{ if .CONSUL_AGENT }}localhost{{ else }}{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}{{ end }}:8500",
-  "preStart": "/usr/local/bin/reload.sh preStart",
-  "logging": {"level": "DEBUG"},
+  "preStart": "generate-config",
   "services": [
     {
       "name": "nginx",
       "port": 80,
-      "health": "/usr/bin/curl --fail --silent --show-error --output /dev/null http://localhost/nginx-health",
+      "health": "curl -fsSo /dev/null http://localhost/nginx-health",
       "poll": 10,
       "ttl": 25,
       "interfaces": ["eth0"]
     },
     {
       "name": "nginx-public",
       "port": 80,
-      "health": "/usr/bin/curl --fail --silent --show-error --output /dev/null http://localhost/nginx-health",
+      "health": "curl -fsSo /dev/null http://localhost/nginx-health",
       "poll": 10,
       "ttl": 25,
       "interfaces": ["eth1", "eth0"]
     }{{ if .ACME_DOMAIN }},
     {
+      "name": "nginx-ssl",
+      "port": 443,
+      "health": "acme init && curl -kfsSo /dev/null https://localhost/nginx-health",
```

Comment:
Ah, it's the staging bit that I was forgetting here. Does LE have a recommended way to test the staging cert? Like maybe if there's a signing root that we can include in our container we can test locally safely?

> Handling all these cases may be complex enough to warrant moving the test into a script instead of cramming all the branches into containerpilot.json, thoughts?

That's probably not a bad idea if we need to implement a lot of logic. We've found that tends to be the case for a lot of the blueprints as we start probing all their edge cases!